evennia/objects/objects.py

```diff
@@ -205,6 +205,12 @@ class DefaultObject(ObjectDB, metaclass=TypeclassBase):
 
     """
 
+    # Determines which order command sets begin to be assembled from.
+    # Objects are usually third.
+    cmd_order = 100
+    cmd_order_error = 100
+    cmd_type = "object"
+
```

Comment:
I feel like sneaking this into a PR that ostensibly is for refactoring/cleanup of the cmdhandler is a bit iffy :wink:  This is a HUGE change to the cmdset merge process. CmdSets already have merge priority and if you want commands from certain cmdsets to merge ahead of others, you change the merge priority of the cmdset. If you want an object's commands to override others, you increase the prio of that cmdset. It's not something you set on the object level.

Unless I'm missing something here, this should be removed.

Reply:
This mechanism is used for sorting the order in which the refined cmdhandler uses to do the merging. like, it first checks session, then account, object. Since the dictionary generated by `get_cmdset_providers` has no concept of order.